sources/packages/backend/apps/workers/src/controllers/assessment/_tests_/e2e/assessment.controller-workflowWrapUp.e2e-spec.ts

```diff
@@ -136,6 +137,13 @@ describe("AssessmentController(e2e)-workflowWrapUp", () => {
         },
       },
     );
+    // Student appeal for the impacted application and its student assessment
+    const studentAppeal = await createFakeStudentAppeal({
+      application: impactedApplication,
+      studentAssessment: impactedApplication.currentAssessment,
+    });
+    impactedApplication.currentAssessment.studentAppeal = studentAppeal;
```

Comment:
Would be below not be enough to create the appeal and the relationship? If not, please disregard it.
```ts
// Student appeal for the impacted application and its student assessment
impactedApplication.currentAssessment.studentAppeal =
  createFakeStudentAppeal({
    application: impactedApplication,
  });
```

Reply:
I wanted to assert the `studentApplea.id` at the end of the test case, so I declared the variable `studentAppeal`.

Reply:
Maybe my point was not clear, what I meant was if the removing the below would not cause any issues.
![image](https://github.com/bcgov/SIMS/assets/61259237/29c976ab-6ede-4060-b9a9-12df2c2071cc)

Reply:
This will work.